src/coord/src/sink_connector.rs

```diff
@@ -20,19 +20,24 @@ use dataflow_types::{
 };
 use expr::GlobalId;
 use ore::collections::CollectionExt;
+use timely::progress::Antichain;
 
 pub async fn build(
     builder: SinkConnectorBuilder,
+    with_snapshot: bool,
+    frontier: Antichain<u64>,
```

Comment:
just want to sanity check myself: is there a circular dependency reason we can't do `Antichain<Timestamp>` here?

Reply:
nope -- looks like I can swap it out!